storage/container_storage_interface/persistent-storage-csi-manila.adoc

```diff
@@ -12,17 +12,18 @@ toc::[]
 Familiarity with link:https://kubernetes.io/docs/concepts/storage/persistent-volumes/[PVs], link:https://kubernetes.io/docs/concepts/storage/persistent-volumes/#persistentvolumeclaims[persistent volume claims (PVCs)], link:http://blog.kubernetes.io/2016/10/dynamic-provisioning-and-storage-in-kubernetes.html[dynamic
 provisioning], and link:https://kubernetes.io/docs/admin/authorization/rbac/[RBAC authorization] is recommended.
 
-Before PVCs can be created, you must install the Manila CSI Driver Operator. The Operator creates the required storage classes for all available Manila share types needed for dynamic provisioning.
+To create CSI-provisioned PVs that mount to Manila storage assets, {product-title} installs the Manila CSI Driver Operator and the Manila CSI driver by default on any OpenStack cluster that has the Manila service enabled.
 
-After the Operator is installed, you must also create the ManilaDriver Custom Resource (CR) that is required in the {product-title} cluster.
+* The **Manila CSI Driver Operator** creates the required storage class that is needed to create PVCs for all available Manila share types. The Operator is installed in the `openshift-cluster-csi-drivers` namespace.
 
-include::modules/persistent-storage-csi-manila-install-operator.adoc[leveloffset=+1]
+* The **Manila CSI driver** enables you to create and mount Manila PVs. The driver is installled in the 'openshift-manila-csi-driver' namespace.
 
-include::modules/persistent-storage-csi-manila-install-driver.adoc[leveloffset=+1]
+== About CSI
```

Comment:
This should probably be in a separate module. Especially since this same text seems to be used in multiple assemblies.

Reply:
Resolved. 

I'd considered making it a separate module before but dropped the idea because the content was so brief. But there is enough for it to stand alone, so I agree modularizing here makes more sense.